src/datepicker/bs-daterangepicker-inline.component.ts

```diff
@@ -55,6 +56,10 @@ export class BsDaterangepickerInlineDirective implements OnInit, OnDestroy, OnCh
      * Disable specific dates
      */
     @Input() datesDisabled: Date[];
+    /**
+     * Predefined ranges
+     */
+    @Input() ranges: BsCustomDates[];
```

Comment:
please push this to component via `config`, let's not bloat components API
thanks in advance!

Reply:
Yes, removed this and moved this to load from config.